src/solver/conv_ocl_dir2Dfwd.cpp

```diff
@@ -484,12 +489,191 @@ ConvSolution ConvOclDirectFwd::GetSolution(const ConvolutionContext& ctx,
     return result;
 }
 
-ConvSolution ConvOclDirectFwdFused::GetSolution(const ConvolutionContext& ctx,
-                                                const ProblemDescription& problem,
-                                                const LegacyPerformanceConfig& config) const
+namespace fusion {
```

Comment:
I would suggest moving fused solver to a separate file.

Reply:
Done